docs/sdk-and-tools/erdjs/erdjs-signing-providers.md

```diff
@@ -43,7 +43,8 @@ The following provider URLs [are defined](https://github.com/ElrondNetwork/elron
 Then, ask the user to log in:
 
 ```
-await provider.login({ callbackUrl: "http://my-dapp" });
+const callbackUrl = encodeURIComponent("http://my-dapp");
+await provider.login({ callbackUrl: callbackUrl });
```

Comment:
could be just `await provider.login({ callbackUrl });`

Reply:
Fixed.